requirements.txt

```diff
@@ -1,6 +1,6 @@
 # project requirements, install them using following command:
 # pip install -r requirements.txt
-lxml
+lxml[html_clean]>=5.2.0
```

Comment:
`lxml[html_clean]` should be fine here, it will keep the older lxml version installed and just print a warning.

Reply:
TIL, for some reason I thought this would raise an error